core/new-gui/src/app/workspace/component/workspace.component.ts

```diff
@@ -121,7 +121,7 @@ export class WorkspaceComponent implements AfterViewInit, OnDestroy {
       .pipe(debounceTime(100))
       .pipe(untilDestroyed(this))
       .subscribe(() => {
-        if (this.userService.isLogin()) {
+        if (this.userService.isLogin() && this.workflowActionService.isWorkflowPersistEnabled()) {
```

Comment:
I just noticed this `workflowPersistEnabled` flag is in `WorkflowActionService`. I think it might be better to be placed at `WorkflowPersistService` as it is more related?

Reply:
agreed. done